R/app_server.R

```diff
@@ -125,15 +125,45 @@ app_server <- function(input, output, session) {
 
   observeEvent(data_r$click, {
     callModule(mod_DT_server, "DT_ui_participants",
-      data = data_r$participants %>%
-        dplyr::select(-dplyr::any_of(getOption("emdash.cols_to_remove_from_participts_table"))) %>%
-        data.table::setnames(originalColumnNames, new_column_names, skip_absent = TRUE)
+               data = data_r$participants %>%
+                 dplyr::select(-dplyr::any_of(getOption("emdash.cols_to_remove_from_participts_table"))) %>%
+                 data.table::setnames(originalColumnNames, new_column_names, skip_absent = TRUE)
     )
     callModule(mod_DT_server, "DT_ui_trips",
-      data = data_r$trips %>%
-        dplyr::select(-dplyr::any_of(getOption("emdash.cols_to_remove_trips_table"))) %>%
-        sf::st_drop_geometry()
+               data = data_r$trips %>%
+                 dplyr::select(-dplyr::any_of(getOption("emdash.cols_to_remove_trips_table"))) %>%
+                 sf::st_drop_geometry()
     )
+    
+    tableList <- getOption('emdash.supplementary_tables')
+
+    # For each supplementary table, append a new tabPanel and run the server function
+    # that specifies table behavior
+    for (t in tableList){
```

Comment:
You may want to look into how we can keep the plot panel above in sync with the table below.